CHANGELOG.md

```diff
@@ -11,6 +11,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Fix switching to master when we're already on master
 - Add MacOS testing
 
+## 2.0.1 (2020-05-21)
```

Comment:
Please always add new changes to the `unreleased` section, as our scripts will do the version bumping for us automatically! 

Thanks :)

Reply:
Shoot, I didn't realize that. Sorry.